frontend/src/components/HomePage/Profiles/profiles.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styles from './profiles.module.css';
+import pfpPlaceholder from '../../Assets/pfpPlaceholder.png';
+import locationIcon from '../../Assets/locationIcon.png'
+import { Link } from 'react-router-dom';
+
+function UserProfile (user) {
```

Comment:
oh this is wrong, my bad for telling you to do it like this but it should instead look like this:

```suggestion
function UserProfile ({ user }) {
```

this is because the default parameter for every react component is a javascript object and what we're doing here is destructuring those properties into usable variables.

https://react.dev/learn/passing-props-to-a-component
https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0